queries/query_template.py

```diff
@@ -0,0 +1,105 @@
+import logging
+import pandas as pd
+from db_manager.augur_manager import AugurManager
+from app import celery_app
+from cache_manager.cache_manager import CacheManager as cm
+import io
+
+"""
+TODO:
+(1) update QUERY_NAME
+(2) update NAME_query in function definition and ack func=, should be the same as file name
```

Comment:
I don't know what this means

Reply:
It is saying to update NAME_query in line 29 and 99 (wouldnt want to base it off of line numbers in the actual description). Lmk what would be more interpretable description

Reply:
maybe
"update 'NAME_query' found in function definition and in the function call that sets the 'ack' variable below. 'NAME' should be the same as filename."